src/bin/inx-chronicle/stardust_inx/mod.rs

```diff
@@ -235,6 +245,26 @@ impl InxWorker {
                     .insert_protocol_parameters(start_index, protocol_parameters)
                     .await?;
             }
+
+            // TODO: This is for migration purposes only. Remove it in the next version release.
```

Comment:
I think we should be able to use `#[deprecated(since = "1.0.0-beta.32")` here to enforce this.

Reply:
This would cause a warning we would also need to ignore. Also, it's the wrong usage.

Reply:
I created an issue instead.